src/vector/v.what.rast.label/v.what.rast.label.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
+##############################################################################
+#
+# MODULE:       v.what.rast.label
+# AUTHOR(S):    Paulo van Breugel <paulo at ecodiv dot earth>
+# PURPOSE:      Upload raster values and labels at positions of vector points
+#               to the vector attribute table.
+#
+# COPYRIGHT: (C) 2016-2022 by Paulo van Breugel and the GRASS Development Team
+#
+#        This program is free software under the GNU General Public
+#        License (>=v2). Read the file COPYING that comes with GRASS
+#        for details.
+##############################################################################
+
+#%module
+#% description: Uploads raster values and labels to vector point layer
+#% keyword: vector
+#% keyword: sampling
+#% keyword: raster
+#% keyword: position
+#% keyword: querying
+#% keyword: attribute table
+#% keyword: surface information
+#%end
+
+#%option G_OPT_V_MAP
+#% key: vector
+#% description: Name vector points map for which to add raster values & labels
+#% guisection: Input
+#% required: yes
+#%end
+
+#%option G_OPT_R_INPUTS
+#% key: raster
+#% description: Name of raster map(s) with labels to be queried
+#% guisection: Input
+#% required: yes
+#% multiple: yes
+#%end
+
+#%option G_OPT_R_INPUTS
+#% key: raster2
+#% description: Name of raster map(s) without labels to be queried
+#% guisection: Input
+#% required: no
+#% multiple: yes
+#%end
+
+#%option G_OPT_V_OUTPUT
+#% description: Name of output point layer
+#% key_desc: name
+#% guisection: Input
+#% required: yes
+#%end
+
+#%flag
+#% key: o
+#% description: Include columns of input vector map
+#%end
+
+#%flag
+#% key: c
+#% description: Include point coordinates
+#%end
+# import libraries
+import os
+import sys
+import uuid
+import atexit
+from subprocess import PIPE
+import textwrap
+import grass.script as gs
+from grass.pygrass.modules import Module
+
+# create set to store names of temporary maps to be deleted upon exit
+clean_layers = []
+
+
+def create_unique_name(name):
+    """Generate a tmp name which contains prefix
+    Store the name in the global list.
+    """
+    return name + str(uuid.uuid4().hex)
+
+
+def tmpname(prefix):
+    """Generate a tmp name which contains prefix
+    Store the name in the global list.
+    """
+    tmpf = create_unique_name(prefix)
+    clean_layers.append(tmpf)
+    return tmpf
```

Comment:
Since GRASS 7.6 there is: https://grass.osgeo.org/grass76/manuals/libpython/script.html?highlight=tempname#script.core.tempname

Reply:
Thanks, implemented